src/obr/core/queries.py

```diff
@@ -94,7 +139,7 @@ def execute_query(query: Query, key, value, latest_only=True, track_keys=list) -
     return query
 
 
-def flatten_jobs(jobs: OpenFOAMProject) -> dict:
+def flatten_jobs(jobs: "OpenFOAMProject") -> dict:
```

Comment:
What is the reasoning for this?

Reply:
Leaving the quotation marks would result in a circular import error, even despite the `if TYPE_CHECKING` clause at the top of the file.